### DISCUSSION TITLE
zigbee2mqtt does not start - but adapter is working in deconz

### DISCUSSION THREAD

**Discussion by GerhardLang** [Koenkk/zigbee2mqtt#8643] [General]
Hello. I hope somebody helps a zigbee/docker newbee;

My Synology has successfully recognized the Conbee2 with deconz. I deactivated deconz, so I do not have any interference.

zigbee2mqtt complains not to find the adapter, herdsman does not start.
Adapter has crw rights for root. Synology docker starts containers in root (like the deconz container).

Do you have any hint for me?

My compose file is:
 version: '3'
  services:
    zigbee2mqtt:
      container_name: zigbee2mqtt
      image: koenkk/zigbee2mqtt
      volumes:
        - /volume1/docker/zigbee2mqtt/data:/app/data
        - /run/udev:/run/udev:ro
      devices:
        - /dev/ttyACM0:/dev/ttyACM0
      restart: always
      network_mode: host
      privileged: true
      environment:
        - TZ=Europe/Amsterdam

Thanks a lot

Gerhard

**Comment by Smo-RBR** [comment]
erial:
  port: /dev/ttyACM0
  adapter: deconz
